src/RecoveryServices/RecoveryServices/help/Wait-AzRecoveryServicesBackupJob.md

```diff
@@ -19,37 +20,31 @@ Wait-AzRecoveryServicesBackupJob [-Job] <Object> [[-Timeout] <Int64>] [-VaultId
 ```
 
 ## DESCRIPTION
+
 The **Wait-AzRecoveryServicesBackupJob** cmdlet waits for an Azure Backup job to finish.
 Backup jobs can take a long time.
 If you run a backup job as part of a script, you may want to force the script to wait for job to finish before it continues to other tasks.
 A script that includes this cmdlet can be simpler than one that polls the Backup service for the job status.
-Set the vault context by using the Set-AzRecoveryServicesVaultContext cmdlet before you use the current cmdlet.
+Set the vault context by using the **Set-AzRecoveryServicesVaultContext** cmdlet before you use the current cmdlet.
```

Comment:
same comment

Reply:
agree. I've changed note regarding vault context and mentioned VaultId parameter there.